.github/workflows/tests.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       matrix:
         os:
           - ubuntu-latest
-        ruby-version: [2.6, 2.7]
+        ruby-version: [2.6, 2.7, 3.0.0-preview1]
```

Comment:
Could you add `3.0.3` and 3.1.0` here pls? I think both are on GHA now, and there's differences between 3.0 and 3.1 with this stuff.

Reply:
OK, that's weird. You can see that calling out 3.1 didn't work when I first tried it (https://github.com/KoanHealth/propono/runs/4874203610?check_suite_focus=true), but ruby setup indicates that 3.1 is supported. 

Let me take another look at that.

Reply:
I have updated the tests.yml to test the correct versions (biggest deal was changing to ruby-setup@v1)